examples/rpc-kitchen-sink/Makefile

```diff
@@ -0,0 +1,45 @@
+.DEFAULT_GOAL := build
+
+ROOT=../..
+REACH=$(ROOT)/reach
+include $(ROOT)/VERSION
+include $(ROOT)/DEPS
+
+NAME="reach-app-rpc-kitchen-sink-client"
+CIMG="reachsh/$(NAME):latest"
+RKEY=REACH_RPC_KEY=$$(cat REACH_RPC_KEY.txt)
+PORT=REACH_RPC_PORT=3001
+
+.PHONY: clean-client-lib-js
+clean-client-lib-js:
+	[ -d ./lib ] && rm -r ./lib || true
+
+.PHONY: clean
+clean: clean-client-lib-js
+	rm -rf build/*.main.mjs
+
+.PHONY: client-lib-js
+client-lib-js: clean-client-lib-js
+	cp -r $(ROOT)/rpc-client/js ./lib
+
+build/index.main.mjs: $(shell find . -name "*.rsh")
+	$(REACH) compile index.rsh main
+
+.PHONY: build
+build: client-lib-js build/index.main.mjs
+	docker build \
+	  --tag=$(CIMG) \
+	  --build-arg NODE_VERSION=$(NODE_VERSION) \
+	  .
+
+run:
+	$(RKEY) $(PORT) $(REACH) rpc-run docker run --rm \
```

Comment:
Part of what I like about the simplicity of `rpc-run` is that you don't need to have hard-coded the keys and ports and stuff. Similarly, my assumption, which I realize now was weird, is that you'd `$(REACH) rpc-run $(REACH) run`, which is definitely strange :)